clang/test/dpct/helper_files_ref/include/dpl_extras/dpcpp_extensions.h

```diff
@@ -617,6 +625,10 @@ void segmented_reduce(sycl::queue queue, T *inputs, T *outputs,
   sycl::range<1> local_size(GROUP_SIZE);
 
   queue.submit([&](sycl::handler &cgh) {
+#ifdef SYCL_EXT_ONEAPI_USER_DEFINED_REDUCTIONS
```

Comment:
Could we not use extra mem for sycl native op?

Reply:
Okay, I'll optimize this, this extra memory is useless for sycl native operator.